Airwallex/Card/Internal/AWXSecurityService.m

```diff
@@ -52,9 +53,9 @@ - (void)doProfile:(NSString *)intentId
 #if TARGET_OS_SIMULATOR
     completion([UIDevice currentDevice].identifierForVendor.UUIDString);
 #else
-    NSString *fraudSessionId = [NSString stringWithFormat:@"%@%f", intentId, [[NSDate date] timeIntervalSince1970]];
-    NSString *sessionId = [NSString stringWithFormat:@"%@%@", AWXCyberSourceMerchantID, fraudSessionId];
-    [self.profiling profileDeviceUsing:@{@"session_id": sessionId} callbackBlock:^(NSDictionary *result) {
+    double timestamp = [[NSDate date] timeIntervalSince1970] * 1000;
+    NSString *fraudSessionId = [NSString stringWithFormat:@"%@%.0f", intentId, timestamp];
```

Comment:
Do we need to pass session id to the request to PA?

Reply:
Yeah, it's already passed to PA as it is. It's named `device_id` when passing to PA.